app/src/main/java/com/keylesspalace/tusky/util/ListStatusAccessibilityDelegate.kt

```diff
@@ -60,7 +62,8 @@ class ListStatusAccessibilityDelegate(
                         R.id.action_open_media_2,
                         R.id.action_open_media_3,
                         R.id.action_open_media_4)
-                for (i in 0 until status.attachments.size) {
+                val attachmentCount = min(status.attachments.size, MAX_MEDIA_ATTACHMENTS)
```

Comment:
So we will add actions only for first 4 attachments right? Seems legit.

Reply:
Yes we also show only 4 images. Maybe we should extend the number sometime.